.travis.yml

```diff
@@ -43,6 +46,8 @@ before_install:
     - if [ "$TRAVIS_BRANCH" = "master" ]; then export COMPOSER_ROOT_VERSION=dev-master; else export COMPOSER_ROOT_VERSION="$TRAVIS_BRANCH".x-dev; fi;
 
 install:
+    - if [ "$twig2" = "yes" ]; then sed -i 's/~1.20|~2.0/~2.0@dev/g' composer.json; composer update --prefer-source; fi;
+    - if [ "$twig2" = "no" ]; then sed -i 's/~1.20|~2.0/~1.20/g' composer.json; composer update --prefer-source; fi;
```

Comment:
we should have some builds running with unmodified deps IMO (deps=high and deps=low will already cover 2.0 and 1.20 btw)